src/simulation/BindingSimulator2D.ts

```diff
@@ -378,11 +301,36 @@ export default class BindingSimulator implements IClientSimulatorImpl {
                 this.currentNumberBound--;
             }
         }
-        // reset to zero for every tenth time point
-        if (this.currentFrame % 10 === 0) {
-            this.currentNumberOfBindingEvents = 0;
-            this.currentNumberOfUnbindingEvents = 0;
+    }
+
+    private resolveChildPositions() {
```

Comment:
this is the big change: before I was using the "isTrigger" flag to keep the detect-collisions library from doing the overlap check if the spheres were bound, and then doing my own collision detection of the complex. But now, I'm just manually moving the child back in place after the system has resolved. This means the bound ligands can sometimes overlap each other but it's not super serious, and better than how it looked before